Makefile

```diff
@@ -249,7 +254,6 @@ eksctl-image: ## Build the eksctl image that has release artefacts and no build
 	$(MAKE) -f Makefile.docker $@
 
 ##@ Site
-
 .PHONY: serve-pages
 serve-pages: ## Serve the site locally
```

Comment:
```suggestion
serve-pages: install-site-deps ## Serve the site locally
```

Reply:
If you add this here (and for the `build-pages` target too), the user won't have to run `make install-site-deps` manually. Just an idea.

Reply:
Thanks for your idea, however, I feel like the number of times you are running _build_ or _serve_ are significantly higher than _install-site-deps_. Additionally, updating depencies are kind of concious action. So even this step is fast, I still would love to keep it sepaprately.

This is kind of consistent with our binary build process as well.

Reply:
Ok, wfm.